shared/src/commonMain/kotlin/co/touchlab/kampkit/sqldelight/CoroutinesExtensions.kt

```diff
@@ -1,49 +1,16 @@
 package co.touchlab.kampkit.sqldelight
 
-import co.touchlab.stately.freeze
 import com.squareup.sqldelight.Query
 import com.squareup.sqldelight.Transacter
 import com.squareup.sqldelight.TransactionWithoutReturn
 import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.channels.Channel
-import kotlinx.coroutines.channels.Channel.Factory.CONFLATED
 import kotlinx.coroutines.flow.Flow
-import kotlinx.coroutines.flow.flow
 import kotlinx.coroutines.flow.map
 import kotlinx.coroutines.flow.mapNotNull
 import kotlinx.coroutines.withContext
 import kotlin.coroutines.CoroutineContext
-import kotlin.jvm.JvmName
 import kotlin.jvm.JvmOverloads
 
-/**
- * Turns this [Query] into a [Flow] which emits whenever the underlying result set changes.
- *
- * Note: We copied this from the sqldelight repo because we're using the new native coroutines draft version.
- * When that is live, we'll be using the official sqldelight version.
- */
-@JvmName("toFlow")
-internal fun <T : Any> Query<T>.asFlow(): Flow<Query<T>> = flow {
-    emit(this@asFlow)
-
-    val channel = Channel<Unit>(CONFLATED).freeze()
-    val listener = object : Query.Listener {
-        override fun queryResultsChanged() {
-            channel.offer(Unit)
-        }
-    }
-
-    addListener(listener)
-    try {
-        for (item in channel) {
-            emit(this@asFlow)
-        }
-    } finally {
-        println("****** Closing Flow in removeListener ******")
-        removeListener(listener)
-    }
-}
-
 @JvmOverloads
 internal fun <T : Any> Flow<Query<T>>.mapToOne(
```

Comment:
Any reason we're keeping this?

Reply:
No, we can dump those too.

Reply:
Removed.

Reply:
I think we can get rid of mapToList too, as it's in the standard extension: https://github.com/cashapp/sqldelight/blob/master/extensions/coroutines-extensions/src/commonMain/kotlin/com/squareup/sqldelight/runtime/coroutines/FlowExtensions.kt#L93. `transactionWithContext` is a different story, though

Reply:
Gone.